### ISSUE TITLE
Installer does not let you install consecutively [Steam Deck / Linux]

### ISSUE THREAD

**Issue by drojf** [07th-mod/python-patcher#187]
A user installed Onikakushi on their Steam Deck, then tried to install Watanagashi (without restarting the installer), but when clicking the "start the installation" button, nothing happened.

See below install log. It looks like it got the message that the button was pressed, but nothing happened. 

The user restarted the installer and could then install Watanagashi, so it's probably some issue when you install one chapter after the other without restarting the installer.

[MOD-INSTALLER-LOG-2022-11-02_18-28-57(1).zip](https://github.com/07th-mod/python-patcher/files/9925315/MOD-INSTALLER-LOG-2022-11-02_18-28-57.1.zip)

**Comment by drojf** [comment]
I did some testing (on a linux machine I was testing something else on), and it looked like the issue was that the install had started, but the UI had not updated yet (when the install starts, it greys out the install settings, then scrolls to the top of the page and shows the install progress).

Notably my internet connection was flaky while doing these tests. Perhaps not having flaky internet just as the test starts causes problems.

Gotta figure out the exact logic for this to happen, as the logs weren't entirely clear.

I'm not certain if the issue I saw was the same as what was reported, as I could eventually install again, even without restarting the installer.
